packages/skills/fipa_negotiation/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
```

Comment:
The core thing to notice with the skills is that they are written so that the fipa_negotiation works completely independent of tac and vice versa. That is we can run tac with other protocols/skills/markets and similarly reuse them in settings outside tac.